spec/traits.dd

```diff
@@ -192,6 +193,12 @@ $(H2 $(GNAME isNested))
     $(DDSUBLINK spec/struct, nested, structs), and
     $(DDSUBLINK spec/function, variadicnested, functions).)
 
+$(H2 $(GNAME isFuture))
+
+    $(P Takes one argument. It returns `true` if the argument is a symbol
+    marked with the `@future` keyword, otherwise `false`. Currently, only
+    functions and variable declarations have support for the `@future` keyword.)
```

Comment:
I think it's worth noting that this is an experimental feature. To the point that it's not even `@future` yet, but `@__future` (IIRC). Also, I think we should strongly state that this is not intended for user code yet, and link to the DIP.

I think this feature is having too much visibility and too little clarity for an experimental feature...